examples/misc/upload_image_asset.py

```diff
@@ -23,20 +23,24 @@
 
```

Comment:
In https://github.com/googleads/google-ads-python/pull/934 I moved this file to the `assets` dir, so just double-check that you're modifying the right file in this PR.